### PR TITLE
ammened backers in readme.md

### LINKED ISSUE #14441: I  readme, the image showing backer is broken,
And maybe we could add some stats about backers too, for people not opeing open collective link, showing it’s something many people use

### PR DESCRIPTION
**Pull request by criticalAY**
## Purpose / Description
Broken link for open collective backers

## Fixes
Fixes #14441

## How Has This Been Tested?

image

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by criticalAY** [comment]
@Arthur-Milchior review request

**Review by david-allison — CHANGES_REQUESTED** [review]
We don't want to hardcode this if at all possible. How do other repos do this?

**Comment by criticalAY** [comment]
> We don't want to hardcode this if at all possible. How do other repos do this?

Webpack [link](https://github.com/webpack/webpack) does it this way it is actually not hardcoded it shows the backers from the backers list that is available on OC and the user is not hardcoded here

**Comment by criticalAY** [comment]
eslint has actually hardcoded their selective sponsors 
```
<!--sponsorsstart-->
<h3>Platinum Sponsors</h3>
<p><a href="#"><img src="https://images.opencollective.com/2021-frameworks-fund/logo.png" alt="Chrome Frameworks Fund" height="undefined"></a> <a href="https://automattic.com"><img src="https://images.opencollective.com/automattic/d0ef3e1/logo.png" alt="Automattic" height="undefined"></a></p><h3>Gold Sponsors</h3>
<p><a href="https://engineering.salesforce.com"><img src="https://images.opencollective.com/salesforce/ca8f997/logo.png" alt="Salesforce" height="96"></a> <a href="https://www.airbnb.com/"><img src="https://images.opencollective.com/airbnb/d327d66/logo.png" alt="Airbnb" height="96"></a></p><h3>Silver Sponsors</h3>
<p><a href="https://liftoff.io/"><img src="https://images.opencollective.com/liftoff/5c4fa84/logo.png" alt="Liftoff" height="64"></a> <a href="https://americanexpress.io"><img src="https://avatars.githubusercontent.com/u/3853301?v=4" alt="American Express" height="64"></a></p><h3>Bronze Sponsors</h3>
<p><a href="https://themeisle.com"><img src="https://images.opencollective.com/themeisle/d5592fe/logo.png" alt="ThemeIsle" height="32"></a> <a href="https://www.crosswordsolver.org/anagram-solver/"><img src="https://images.opencollective.com/anagram-solver/2666271/logo.png" alt="Anagram Solver" height="32"></a> <a href="https://icons8.com/"><img src="https://images.opencollective.com/icons8/7fa1641/logo.png" alt="Icons8" height="32"></a> <a href="https://discord.com"><img src="https://images.opencollective.com/discordapp/f9645d9/logo.png" alt="Discord" height="32"></a> <a href="https://transloadit.com/"><img src="https://avatars.githubusercontent.com/u/125754?v=4" alt="Transloadit" height="32"></a> <a href="https://www.ignitionapp.com"><img src="https://avatars.githubusercontent.com/u/5753491?v=4" alt="Ignition" height="32"></a> <a href="https://herocoders.com"><img src="https://avatars.githubusercontent.com/u/37549774?v=4" alt="HeroCoders" height="32"></a> <a href="https://quickbookstoolhub.com"><img src="https://avatars.githubusercontent.com/u/95090305?u=e5bc398ef775c9ed19f955c675cdc1fb6abf01df&v=4" alt="QuickBooks Tool hub" height="32"></a></p>
<!--sponsorsend-->
```

**Comment by lukstbit** [comment]
I think we could also use their API and create a workflow to automatically update the readme file.

**Comment by criticalAY** [comment]
> I think we could also use their API and create a workflow to automatically update the readme file.

But why to complicate things when we can simply use the URL in here and its updates the user from the backers list

**Comment by lukstbit** [comment]
I just mentioned the API as an alternative, I haven't checked to see if this would actually work.

The problem with your approach is that it's manual, so any new backers would need to be added manually especially as we mention that backing the project would get their name on our readme. So someone would have to go pretty often and check if there are new backers(I see a big list) and then add them in the readme(with a long list of commits "Add new backers").

**Comment by criticalAY** [comment]
Umm webpack has actually mentioned that if a backers name is not there they need to create an issue and they would be added just in case but I'll see if there is a API available(not sure there is )

**Comment by lukstbit** [comment]
> Umm webpack has actually mentioned that if a backers name is not there they need to create an issue

The code in this PR + a text added that if you are a backer and you don't see your name in the list to create an issue sounds like a simple solution.

**Comment by criticalAY** [comment]
Ok will do

**Comment by lukstbit** [comment]
I think the text should be something like(also remove the empty row above this text):

_If you can't see yourself here, please let us know by creating an issue._

but wait for other reviewers' opinions as well. Note that when I looked at this from the files tab the backers images are not seen:

Screenshot from 2023-10-09 12-02-28

**Comment by criticalAY** [comment]
I can see the list i tested it before pushing it
image

**Comment by eginhard** [comment]
It already shows the top, not first, 100 backers, so I assume OpenCollective would update the linked images. There are currently 3584 total backers, so it surely is not feasible to list them all in this way.

**Comment by criticalAY** [comment]
Can we add around 50 backers as I added top 100 and then give a link to more show them all

**Comment by criticalAY** [comment]
@lukstbit @david-allison I figured it out we can display all the backers without hardcoding anything I'll update the PR once I'm home

**Comment by criticalAY** [comment]
i open an issue for the same in OC repo [here](https://github.com/opencollective/opencollective/issues/7046)

**Comment by criticalAY** [comment]
@lukstbit @david-allison 
We can't render all the 3k backers due to the SVG size,
we have two options here
1. Just show the number of backers badge(current commit)
2. Show 1k backers along with this badge

**Comment by david-allison** [comment]
I prefer option 1. We could get in contact with Open Collective and add some backlog work for a more feasible option, but I  wouldn't want to see automated processes here unless there's a requirement. 

* We don't want to spend much time here
* We don't want ongoing maintenance here

Speaking for our backers: I'd much rather their money went to user-facing changes, rather than our readme.

----

This appears broken on your branch: 

https://github.com/criticalAY/Anki-Android/blob/backers-link/README.md

Screenshot:
Screenshot 2023-10-10 at 16 28 10

**Comment by criticalAY** [comment]
updated it
image

File: README.md [hunk 1]
```diff
@@ -96,33 +96,20 @@ Thanks to these awesome code contributors who keep this project going
...
-<a href="https://opencollective.com/ankidroid/sponsor/19/website" target="_blank"><img src="https://opencollective.com/ankidroid/sponsor/19/avatar.svg"></a>
-<a href="https://opencollective.com/ankidroid/sponsor/20/website" target="_blank"><img src="https://opencollective.com/ankidroid/sponsor/20/avatar.svg"></a>
+<a href="https://opencollective.com/ankidroid#sponsor" target="_blank">
+  <img src="https://opencollective.com/Ankidroid/sponsors.svg?width=890" />
+</a>
 
 ### [Backers](https://opencollective.com/ankidroid#backer)
 
-Thank you to all our backers! 🙏
+A big thank you to each of our 🙏
```
[the remark this answers is not in the document]
Comment (criticalAY) [reply README.md:105]: i left that on purpose as the badge already has that text but will amend this

[the remark this answers is not in the document]
Comment (criticalAY) [reply README.md]: Done!

**Review by david-allison — APPROVED** [review]

**Review by lukstbit — APPROVED** [review]
LGTM

**Pull request merged**